docs/setup/upgrade/upgrade-migrations.asciidoc

```diff
@@ -111,17 +94,15 @@ In order to rollback after a failed upgrade migration, the saved object indices
 [float]
 ===== (Not recommended) Rollback without a backup snapshot:
```

Comment:
Maybe creating a backup should be listed as the very first item before starting the migration?

Reply:
The actual step-by-step upgrade instructions are on a different page where we have a warning and it's one of the "before you begin" warnings https://www.elastic.co/guide/en/kibana/current/upgrade.html#upgrade-before-you-begin